Client/Frontend/Settings/NewTabPageTableViewController.swift

```diff
@@ -0,0 +1,59 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import Static
+import BraveShared
+import Shared
+
+class NewTabPageTableViewController: TableViewController {
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        navigationItem.title = Strings.NewTabPageSettingsTitle
+        tableView.accessibilityIdentifier = "NewTabPageSettings.tableView"
+        dataSource.sections = [section]
+        
+        // Sponsored switch is only enabled if the background images is enabled also.
+        self.sponsoredSwitch?.isEnabled = Preferences.NewTabPage.backgroundImages.value
```

Comment:
I don't see sponsored switch anymore, intentional?

Reply:
It only shows in the JP deal regions currently. See https://github.com/brave/brave-ios/pull/1997/files#diff-152faad3f3095d626b3584277fe0f741R43

This will be changed eventually.